src/map.rs

```diff
@@ -380,17 +380,23 @@ where
                     let mut p: Option<&Node<K, V>> = Some(node);
                     while p.is_some() {
                         delta -= 1;
+                        unsafe {
```

Comment:
This should include a safety argument -- see `remove`.

Reply:
Can I just copy the safety note from the `replace_node` method?